llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -16562,11 +16565,11 @@ bool RISCVTargetLowering::isUsedByReturnOnly(SDNode *N, SDValue &Chain) const {
     return false;
 
   SDNode *Copy = *N->use_begin();
-  
+
```

Comment:
Is this upstream's whitespace? We should keep it if so.

Reply:
Okay so it will go away on the next merge without causing merge conflicts. Fine to remove it now.